series.go

```diff
@@ -5,20 +5,25 @@ import (
 	"net/http"
 )
 
+// AddSeries creates a new series in the database.
```

Comment:
Here is missing info about seasons. They could be interpreted as series -> series -> items. It is important part how to distinguish between seasons and create better recommendation for example for new season (series of series) appearing in series container.

Reply:
I would rather push there the reference than copy it.

However, the official documentation is generic. Common use case for series->season->episode tree is to use this call 2 times and I rather mention it as we did the way through than rely on official documentation.